### PR TITLE
Fix akka#21028 - Publish de Akka Stream TestKit utilities

### PR DESCRIPTION
**Pull request by cchantep**
See https://github.com/akka/akka/issues/21028

### CODE REVIEW

File: contrib/src/main/scala/akka/stream/contrib/TestKit.scala [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import akka.actor.ActorRef
+import akka.actor.ActorRefWithCell
+import akka.stream.Materializer
+import akka.stream.impl._
+import akka.testkit.TestProbe
+import com.typesafe.config.ConfigFactory
+import scala.concurrent.duration._
+import scala.util.control.NoStackTrace
+
+object TestKit {
```
Comment (cchantep) [inline contrib/src/main/scala/akka/stream/contrib/TestKit.scala]: Not sure what's the best naming for this one

File: contrib/src/main/scala/akka/stream/contrib/TestKit.scala [hunk 2]
```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.constrib
```
Comment (johanandren) [inline contrib/src/main/scala/akka/stream/contrib/TestKit.scala]: `constrib` -> `contrib`

**Comment by johanandren** [comment]
I think we should take a step back here and consider if we should make the copied test tools a part of the stream testkit rather than duplicating them.

**Comment by drewhk** [comment]
I agree. The Coroner is not streams stuff though but akka-actor-tests. I think for contrib it might be enough to have the assertAllStagesStopped to be exposed. But then again, it might be better to polish that functionality to a more general management facility for running streams. Maybe let's merge this here and remove it once we built better support for introspecting running streams?

File: contrib/build.sbt [hunk 3]
```diff
@@ -4,7 +4,8 @@ lazy val contrib = (project in file(".")).
 name := "akka-stream-contrib"
 
 libraryDependencies ++= Seq(
-  "junit"            %  "junit"           % "4.12" % Test, // Common Public License 1.0
-  "com.novocode"     %  "junit-interface" % "0.11" % Test, // BSD-like
-  "com.google.jimfs" %  "jimfs"           % "1.1"  % Test  // ApacheV2
+  "com.typesafe.akka" %% "akka-stream-testkit" % Common.AkkaVersion,
```
Comment (cchantep) [inline contrib/build.sbt]: For now that's only used there

File: contrib/src/main/scala/akka/stream/contrib/TestKit.scala (hunk 2 above)
Comment (cchantep) [inline contrib/src/main/scala/akka/stream/contrib/TestKit.scala]: Fixed

**Comment by drewhk** [comment]
So we opted for merging right?

**Comment by johanandren** [comment]
Yes, LGTM!

**Comment by cchantep** [comment]
Thanks

**Comment by drewhk** [comment]
Thanks for your contribution!

**Pull request merged**